src/OI.cpp

```diff
@@ -43,7 +47,11 @@ OI::OI()
 
   SmartDashboard::PutData("TankDrive", new TankDrive());
   SmartDashboard::PutData("Autonomous Command", new AutonomousCommand());
-
+  SmartDashboard::PutData("FeederOn", new ConveyorControl(1.0, 0.0));
+  SmartDashboard::PutData("MoverOn", new ConveyorControl(0.0, 1.0));
+  SmartDashboard::PutData("IntakeOn", new SetIntake(1.0));
+  SmartDashboard::PutData("ShooterOn", new SetShooter(1.0));
+  SmartDashboard::PutData("TurretOn", new SetTurret(1.0));
```

Comment:
You cancel the command on the Smart Dashboard. If you haven't seen the buttons, they initially have a Start button next to them, and once click it transforms to a Cancel.